exporter/opensearchexporter/README.md

```diff
@@ -20,9 +20,15 @@ The documents are sent using [observability catalog](https://github.com/opensear
 
 ## Configuration options
 ### Indexing Options
+The Observability indices would follow the recommended for immutable data stream ingestion pattern using
```

Comment:
I'm not sure I understand this sentence. Should this be this?

```suggestion
The Observability indices would follow the recommended pattern for immutable data stream ingestion using
```

Reply:
You are right